src/material-experimental/mdc-chips/testing/chip-harness.ts

```diff
@@ -52,4 +62,19 @@ export class MatChipHarness extends ComponentHarness {
   async getRemoveButton(filter: ChipRemoveHarnessFilters = {}): Promise<MatChipRemoveHarness> {
     return this.locatorFor(MatChipRemoveHarness.with(filter))();
   }
+
+  /**
+   * Whether the chip has an avatar
+   */
+   async hasAvatar() {
```

Comment:
I don't think that we need this since it can be inferred from `getAvatar`.

Reply:
right, removed